sdk/keyvault/azure-keyvault-administration/azure/keyvault/administration/aio/_access_control_client.py

```diff
@@ -107,10 +120,14 @@ def list_role_assignments(self, role_scope: str, **kwargs: "Any") -> "AsyncItemP
         )
 
     @distributed_trace
-    def list_role_definitions(self, role_scope: str, **kwargs: "Any") -> "AsyncItemPaged[KeyVaultRoleDefinition]":
+    def list_role_definitions(
+        self, role_scope: "Union[str, KeyVaultRoleScope]", **kwargs: "Any"
+    ) -> "AsyncItemPaged[KeyVaultRoleDefinition]":
         """List all role definitions applicable at and above a scope.
 
-        :param str role_scope: scope of the role definitions
+        :param role_scope: scope of the role definitions. We've provided enum :class:`KeyVaultRoleScope`
+            to list the well known scopes, but those scopes are not exhaustive
```

Comment:
Including we the SDK developers as actors in the reference docs feels contrary to their style to me. How about something like this to keep the action focused on code objects:
```suggestion
        :param role_scope: scope of the role definitions. :class:`KeyVaultRoleScope` defines common broad scopes. To
            list role definitions for a narrower scope, specify it as a string.
```